Docker/docker-compose-local-mssql.yml

```diff
@@ -0,0 +1,113 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+# This docker compose is only intended for local testing of the dockerfiles in
+# the parent docker/ directory. Those files load the Ed-Fi components from
+# publisehd NuGet packages rather than from source code.
+
+services:
+  db-ods:
+    build:
+      context: ods-api-db-ods-minimal/ubuntu/mssql
+      dockerfile: Dockerfile
+      args:
+        ODS_VERSION: ${MSSQL_ODS_MINIMAL_VERSION}
+        TPDM_VERSION: ${MSSQL_TPDM_MINIMAL_VERSION}
+        STANDARD_VERSION: ${STANDARD_VERSION}
+        EXTENSION_VERSION: ${EXTENSION_VERSION}
+    environment:
+      SQLSERVER_USER: ${SQLSERVER_USER}
+      SQLSERVER_PASSWORD: ${SQLSERVER_PASSWORD}
+      TPDM_ENABLED: ${TPDM_ENABLED:-true}
+    volumes:
+      - vol-db-ods:/var/opt/mssql/data
```

Comment:
We should also mount the volume for `/var/opt/mssql/log` otherwise its data would be loss on container restarts.
Same for the other compose files.

Reply:
Fixed